pkg/cli/admin/upgrade/status/workerpool_test.go

```diff
@@ -967,12 +967,13 @@ func Test_nodeInsights(t *testing.T) {
 	mcpMaster := mcp("master").setMachineConfig("new").machineConfigPool
 	mcpWorker := mcp("worker").setMachineConfig("new").machineConfigPool
 	type args struct {
-		pool          mcfgv1.MachineConfigPool
-		node          corev1.Node
-		reason        string
-		isUnavailable bool
-		isUpdating    bool
-		isDegraded    bool
+		pool             mcfgv1.MachineConfigPool
+		node             corev1.Node
+		reason           string
+		isUnavailable    bool
+		isUpdating       bool
+		isDegraded       bool
+		unavailableSince time.Time
```

Comment:
seems like we added a new possible test input but we're not actually using it (=only testing when it's empty). do you plan to add more tests here to utilize it?